plugins/www/Optimobile.js

```diff
@@ -0,0 +1,8 @@
+
+
+var exec = require('cordova/exec');
+
+
+window.initialize = function (arg0, arg1, success, error) {
```

Comment:
Can we have better arg names, assume arg0 / arg1 are the sets of credentials?

Reply:
the args name were changed to  arg0->optimove_credentials and arg1->optimove_mobile_credentials

Reply:
Point on naming, can we work with `lowerCamelCase` in general for JS/TS files?